packages/cms-lib/hubdb.js

```diff
@@ -23,12 +24,9 @@ async function createHubDbTable(portalId, src) {
   if (path.extname(src) !== '.json') {
     throw new Error('The HubDB table file must be a ".json" file');
   }
+}
 
-  const table = fs.readJsonSync(src);
-  const { rows, ...schema } = table;
-
-  const { columns, id } = await createTable(portalId, schema);
-
+async function updateRows(portalId, tableId, rows, columns) {
```

Comment:
I think that this function name needs to be tweaked since it is adding rows.

Reply:
Changed to `addRows`